README.md

```diff
@@ -46,7 +46,14 @@ var ActiveDirectory = require('activedirectory2');
 var config = { url: 'ldap://dc.domain.com',
```

Comment:
With this change, I'd go ahead and change the whole block to look like:

```js
var config = {
  url: 'foo',
  user: [
    // comment
    'dn'
  ]
}
```

The current reduced format become illegible with the inlined comments.